content/webapp/pages/collections.tsx

```diff
@@ -20,7 +20,9 @@ export const getServerSideProps: GetServerSideProps<
 };
 
 const CollectionsPage: FunctionComponent<page.Props> = (props: page.Props) => {
+  useHotjar(true);
```

Comment:
sneaky

Reply:
I'm slightly unclear as to why it showed up like a change ahha it's in main!